### PR TITLE
update fuse-backend-rs to 0.4.0.

### PR DESCRIPTION
**Pull request by wllenyj**
In the new version of fuse-backend-rs, the
`fuse_backend_rs::abi::linux_abi` mod has been replaced with
`fuse_backend_rs::abi::fuse_abi`.

Signed-off-by: wllenyj <wllenyj@linux.alibaba.com>

### CODE REVIEW

File: src/bin/nydusd/fusedev.rs [hunk 1]
```diff
@@ -97,10 +97,10 @@ struct FuseServer {
 }
 
 impl FuseServer {
-    fn new(server: Arc<Server<Arc<Vfs>>>, se: &FuseSession, evtfd: EventFd) -> Result<FuseServer> {
+    fn new(server: Arc<Server<Arc<Vfs>>>, se: &FuseSession, _evtfd: EventFd) -> Result<FuseServer> {
         Ok(FuseServer {
             server,
-            ch: se.new_channel(evtfd)?,
+            ch: se.new_channel()?,
```
Comment (liubogithub) [inline src/bin/nydusd/fusedev.rs:103]: I thought the patch is to do a plain replacement of `linux_abi`, but this seems irrelevant?

Comment (wllenyj) [reply src/bin/nydusd/fusedev.rs:103]: The `new_channel` function signature has also changed.

**Comment by killagu** [comment]
Duplicated with https://github.com/dragonflyoss/image-service/pull/316.

**Comment by bergwolf** [comment]
yep, I think it is no longer needed after #316 is merged. @wllenyj please close it if that is the case.
